sphinx-docs/conf.py

```diff
@@ -29,6 +30,8 @@ def visit_document(*_):
 apidocs_argv = ['-o', '_generated', '--implicit-namespaces', '--force', str(caldera_root_dir / 'app')]
 apidoc.main(apidocs_argv)
 
+# Import plugin docs
```

Comment:
I'm hesitant to remove it, only because this is treated more as a configuration file than a python file in the examples I've seen. As far as I can tell, most entries in this file should have comments associated with them.